common/src/main/java/com/linecorp/centraldogma/common/Entry.java

```diff
@@ -93,7 +93,20 @@ public static Entry<String> ofText(Revision revision, String path, String conten
      * @param <T> the content type. {@link JsonNode} if JSON. {@link String} if text.
      */
     public static <T> Entry<T> of(Revision revision, String path, EntryType type, @Nullable T content) {
-        return new Entry<>(revision, path, type, content);
+        return new Entry<>(revision, path, type, content, false);
+    }
+
+    /**
+     * Returns a newly-created {@link Entry} whose content is nullable.
+     *
+     * @param revision the revision of the {@link Entry}
+     * @param path the path of the {@link Entry}
+     * @param content the content of the {@link Entry}
+     * @param type the type of the {@link Entry}
+     * @param <T> the content type. {@link JsonNode} if JSON. {@link String} if text.
+     */
+    public static <T> Entry<T> ofNullable(Revision revision, String path, EntryType type, @Nullable T content) {
```

Comment:
What do you think of removing the `content` from the parameters if you want to make a nullable content entry?
Otherwise, I think we can use the above method which takes a `@Nullable T content`.